### PR TITLE
NGSTACK-943 add default alt text for image link and add alt text in o…

### PR DESCRIPTION
**Pull request by baljakl**
…verlay/ng_video

### CODE REVIEW

**Review by Ljudevit — COMMENTED** [review]
I do not believe we want a hardcoded text before alt_text. Alt text can be a proper defined label added in administration which will cover the whole context of it. hardcoded alt text should only be a fallback IF there is not alt_text set and we only have a content.name defined. then a fallback would be "read more about {{ content.name }}".

Also, do not forget about german, we need a translation label for "read more about" in german as well.

File: templates/themes/app/content/views/overlay/ng_video.html.twig [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <article {{ toolbar_macros.item_params(content, location) }} class="view-type view-type-{{ view_type }} ng-video vl2">
         <i class="icon-play article-icon" aria-hidden="true"></i>
         <figure class="image">
-            {{ video.poster(content, location) }}
+            {{ video.poster(content, location, null, content_fields.title(content)) }}
```
Comment (zulicek) [inline templates/themes/app/content/views/overlay/ng_video.html.twig:12]: Why are those two parameters added?

Comment (baljakl) [reply templates/themes/app/content/views/overlay/ng_video.html.twig:12]: It's alias and alt text for video.poster macro.

Comment (Ljudevit) [reply templates/themes/app/content/views/overlay/ng_video.html.twig:12]: i would suggest adding some alias, just to be safe.

Comment (baljakl) [reply templates/themes/app/content/views/overlay/ng_video.html.twig]: added alias in standard and overlay - ng_video

**Review by zulicek — APPROVED** [review]

File: translations/messages.de.yaml [hunk 2]
```diff
@@ -9,6 +9,8 @@ ngsite.layout.cookie_settings.accepted: "Akzeptiert"
 ngsite.layout.cookie_settings.not_accepted: "Nicht akzeptiert"
 ngsite.layout.sponsored: 'Gesponsert'
 
+ngsite.general.read_more_about: 'Mehr erfahren über'
```
Comment (emodric) [inline translations/messages.de.yaml:12]: Please rename this to `ngsite.read_more_about`, since we're using that format in other PRs.

Comment (baljakl) [reply translations/messages.de.yaml]: https://github.com/netgen/media-site/pull/219/commits/9fe1d519565ee64b53de0a0ea5ed3154b9c8bf07

renamed label for translation

**Review by emodric — APPROVED** [review]

**Pull request merged**